func/code.func

```diff
@@ -92,73 +95,90 @@ _ unpack_order(slice order) inline_ref {
 
   slice sender = cs~load_msg_addr();
 
-  (slice stats, cell orders, slice owner1, slice owner2, slice echo) = unpack_state();
+  (slice stats, cell orders, slice owner1, slice owner2, slice ecvrf) = unpack_state();
+
+  ;; subscribe to ecvrf 
```

Comment:
ECVRF subscription is indeed required so lottery can function automatically.

Reply:
Although probably not in this place, since now it's called even on owner's deposit or reception of random number.